<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Facture</class>
 <widget class="QWidget" name="Facture">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1278</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="secondTable">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>961</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="dateLabel">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>50</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>60</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>date</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="companyNameLabel">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>90</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="companyIdLabel">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>130</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="telBox">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>170</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="MatFiscspinBox">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>200</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="addressSpinBox">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>240</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="factureIdLabel">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>290</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="totalHTVALineEdit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>540</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="totalRemiseLineEdit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>570</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="totalTVALineEdit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>610</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="totalTTCLineEdit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>650</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="totalTTCInWordsLabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>680</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addLineButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>addLineButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="finishedButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>90</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>finishedButton</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="timbreFiscalSpinBox">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>480</y>
     <width>42</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="update">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>90</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>updateAmount</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tvaSpinBox">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>540</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="remiseSpinBox">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>540</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>480</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>timbre fiscal</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>540</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>tva</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>550</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>remise</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>968</x>
     <y>90</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>company name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>140</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>id</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>180</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>tel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>200</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>matFisc</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>250</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>address</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>300</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>facture ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>540</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TOTAL htva</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>188</x>
     <y>570</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>total remise</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>610</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>total TVA</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>650</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>total TTC</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>680</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>total TTC in word</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browseButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>browseButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
